scipio/src/executor.rs

```diff
@@ -740,9 +745,18 @@ impl LocalExecutor {
 
         let waker = waker_fn(|| {});
         let cx = &mut Context::from_waker(&waker);
+        let preempt = self.preempt_timer_duration();
+
+        let start = now(CLOCK_MONOTONIC);
+
+        let ru0 = ResourceUsage::runtime();
+        let mut stats = ExecutorStats::new();
 
         LOCAL_EX.set(self, || loop {
-            if let Poll::Ready(t) = future.as_mut().poll(cx) {
+            if let Poll::Ready(t) = LocalExecutor::measure_poll(&mut stats, future.as_mut(), cx) {
```

Comment:
why are we only accounting when the future is ready ?
In a long running server this future will essentially never be ready and we'll always be pulling tasks from the task queue.

Reply:
Wiped